crates/topos-tce-api/src/runtime/mod.rs

```diff
@@ -293,8 +297,9 @@ impl Runtime {
             } => {
                 info!("Stream {stream_id} is registered as subscriber");
 
-                if let Some(task) = self.sync_tasks.get(&stream_id) {
-                    task.abort();
+                if let Some(cancel_token) = self.sync_tasks.remove(&stream_id) {
```

Comment:
What happens if the `sender.send()` call on line 312 fails? Maybe nothing at all, but I just wanted to double check that we understand what that failure condition means and if we're at all worried, consider moving this code here furthe down?